src/settings/ServicePointDetails.js

```diff
@@ -225,76 +236,64 @@ class ServicePointDetails extends React.Component {
+        },
+        [THURSDAY]: {
+          columnTestProp: 'thursday',
+          labelId: 'thuDayShort',
+        },
+        [FRIDAY]: {
+          columnTestProp: 'friday',
+          labelId: 'friDayShort',
+        },
+        [SATURDAY]: {
+          columnTestProp: 'saturday',
+          labelId: 'satDayShort',
+        },
+      };
+
       currentPTimes =
         <Row>
           <Col xs>
             <div
               data-test-service-point-current-period-times
               className="seven-cols"
             >
-              <div
-                data-test-sunday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.sunDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[0])}
-                />
-              </div>
-              <div
-                data-test-monday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.monDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[1])}
-                />
-              </div>
-              <div
-                data-test-tuesday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.tueDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[2])}
-                />
-              </div>
-              <div
-                data-test-wednesday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.wedDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[3])}
-                />
-              </div>
-              <div
-                data-test-thursday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.thuDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[4])}
-                />
-              </div>
-              <div
-                data-test-friday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.friDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[5])}
-                />
-              </div>
-              <div
-                data-test-saturday
-                className="col-sm-1"
-              >
-                <KeyValue
-                  label={<FormattedMessage id="ui-calendar.satDayShort" />}
-                  value={this.getWeekdayOpeningHours(weekdays[6])}
-                />
-              </div>
+              {weekdays.map((weekday) => {
+                const weekdayData = weekdaysData[weekday];
+                const columnTestProp = `data-test-${weekdayData.columnTestProp}`;
```

Comment:
Could you please double check tests?

Reply:
checked. we still need to have `data-test-` attribute. they are used in tests